core/src/main/resources/hudson/logging/LogRecorderManager/index.jelly

```diff
@@ -29,6 +29,7 @@ THE SOFTWARE.
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
 <l:layout title="${%Log}">
   <st:include page="sidepanel.jelly" />
+  <l:breadcrumb title="${%Log Recorders}" />
```

Comment:
🐛 Same as for the plugin manager, this does not provide the expected behavior.

It's working for levels/new but not for index (default page)

<details><summary>Screenshots</summary>

### Before


![Screenshot_2022-11-29_194734_001](https://user-images.githubusercontent.com/2662497/204619082-2fd7c384-a7ad-4c20-9bdc-583496da043b.png)

### After

![Screenshot_2022-11-29_194653_001](https://user-images.githubusercontent.com/2662497/204619039-b4c41322-287a-4289-b709-ccf59706ed33.png)


</details>